### ISSUE TITLE
Publier les données sur data.gouv.fr

### ISSUE THREAD

**Issue by AntoineAugusti** [cpesr/emploiESR#3]
Tout le monde peut publier des données sur https://www.data.gouv.fr/fr/, pourquoi ne pas faire ça avec ces données ? Si le MESRI ne les publie pas (encore ?) en open data, données brutes, c'est une bonne idée.

**Comment by juliengossa** [comment]
Oui, c'est une bonne idée, mais qui demande énormément de travail, notamment pour bien documenter les données. Même le ministère a parfois des problèmes pour le faire correctement. Et généralement, l'usage est très rare ensuite. Donc pour l'instant, ça n'est pas prévu.
